wgsl/index.bs

```diff
@@ -6055,6 +6055,7 @@ A `continue` statement must only be used in a [=statement/loop=] or [=statement/
 A `continue` statement must not be placed such that it would transfer
 control to an enclosing [=statement/continuing=] statement.
 (It is a *forward* branch when branching to a `continuing` statement.)
+i.e. a `continue` must not appear inside a `continuing` block.
```

Comment:
As discussed on chat. This rule allows a continue statement that is within a loop nested in a continuing statement. Maybe a better clarification would be a note to that effect.

Reply:
Is that better or worse?